utils/src/main/kotlin/OrtProxySelector.kt

```diff
@@ -28,6 +28,7 @@ import java.net.SocketAddress
 import java.net.URI
 
 typealias AuthenticatedProxy = Pair<Proxy, PasswordAuthentication?>
+typealias ProxyProtocolMap = Map<String, List<AuthenticatedProxy>>
```

Comment:
I believe `ProtocolProxyMap` would be a better name, because this order reflects that it maps protocols to proxies.

Reply:
Done.